helper/hashcode/hashcode.go

```diff
@@ -1,10 +1,16 @@
 package hashcode
 
-import (
-	"hash/crc32"
-)
+import "hash/crc32"
```

Comment:
Stylewise I usually just prefer to use the parens, because it makes it easy to add imports later, despite being not the most optimal way (visually) to put it.